diagnostic.md

```diff
@@ -6,70 +6,71 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
-// your response here
+get '/books', to: 'books4'
+resources :books, only: [4]
 ```
 
 In what file does the above code go?
 
 ```md
-// your response here
+config/routes.rb
 ```
 
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
-// your response here
+sets up migration
 ```
 
 What is the difference between `rails g scaffold things` and
 `rails g controller things`?
 
 ```md
-// your response here
+scaffold is used when generating code by hand, controller uses rails
```

Comment:
One command will generate everything you need, whereas the other will simply create a controller for you